src/dosext/mfs/mfs.c

```diff
@@ -957,8 +958,11 @@ mfs_redirector(void)
 }
 
 int
-mfs_inte6(void)
+mfs_inte6(int fromDOS)
 {
+  if (fromDOS && WORD(REGS.ebx) == DOS_SUBHELPER_MFS_REDIR_INIT)
+    emufs_loaded = TRUE;
```

Comment:
> I could walk the Device Driver chain from the pointer in LOL?

No, I have not suggested checking it that
way exactly because we do not set vector.

> I put the thing into version check mainly because you suggested it would be a good idea.

I didn't mean version check should return an
error. I just meant an additional status. If you
want to return an error (and I don't see why
you wouldn't want to), then why not to re-use
the existing 0 helper? It served no purpose,
but now it will! :) The good thing about it is
that it may be called only by `emufs.sys` and
no other "initializers" that are still there till we
make `emufs.sys` a mandatory. So it can be
used for both setting the `emufs_loaded` and
returning an error.